### ISSUE TITLE
iOS: 2 Slides causes issues

### ISSUE THREAD

**Issue by erjdriver** [manijak/nativescript-carousel#59]
If I have 2 slides - when sliding forward - the new image scrolls in as the old image leaves the screen (right to left)

When sliding backwards though, the old image scrolls out and in scrolls a blank white page (left to right) - when the old image is completely scrolled off - then the white page is replaced with the new image.

This on the iOS platform.

If I have 3 images - butta...

**Comment by manijak** [comment]
Yeah there still seems to be an issue when there are just two slides present. I am guessing that your are using the Carousel with the `inifinite` mode. Try setting the `finite` property to `true` and see if it works better.

**Comment by erjdriver** [comment]
if I set finite to true - then it works fine.

**Comment by sushant-j** [comment]
I am getting the same error. 
Is there any plan to look into it?

**Comment by manijak** [comment]
Yeah sorry about this. The iOS part for the Carousel plugin is based on a ObjC pod, I have been digging in the pod code trying to figure out what is going on, but failed to find a propper solution. In the meantime I would suggest setting `finite` to `true` if you only have 2 slides.

**Comment by sushant-j** [comment]
Yeah, I am doing that as of now.
